packaging/repoconfig/netdata-deb-repo-priority

```diff
@@ -0,0 +1,3 @@
+Package: *
+Pin: origin repository.netdata.cloud
+Priority: 1000
```

Comment:
What about "repository.netdata.cloud"? I though we were switching to this URL.

Reply:
Yes, and that is the exact URL used here.

Reply:
Ok, what about "repo.netdata.cloud"? Atm we are using it, right?

Reply:
No, the repository configuration packages are already pointing at repository.netdata.cloud.